MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.AppCompat/MvvmCross.Droid.Support.V7.AppCompat.csproj

```diff
@@ -138,6 +139,9 @@
       <Name>MvvmCross.Droid.Support.Core.UI</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
+    <AndroidResource Include="Resources\Xml\Empty.xml" />
```

Comment:
Looking at our sample project, this one misses some lines. But i have no idea if it actually works like that: https://github.com/MvvmCross/MvvmCross/blob/develop/TestProjects/Android-Support/Fragments/Example.Droid/Example.Droid.csproj#L13

Maybe this is even a bug.